scania-digital-design-system/tegel
# feat(table): disabled state of checkbox in multiselect rows

**Describe pull-request**  
Enable user to control disabled state of checkbox in multiselect feature

**Solving issue**  
Fixes: [CDEP-3107](https://tegel.atlassian.net/browse/CDEP-3107)

**How to test**  
1. Open the preview link
2. Go to multiselect story of Table 
3. Try to select "All disabled"
4. Open a branch on your local
5. In the story try to set both the header row and body row with "disabled"
6. Make sure both appear in a disabled state


**Additional context**  
It is a bit tricky to demo in Story disabled state as we would have to repeat it for every single row. I consider it is enough to have it under the "Notes" as one of props. 


[CDEP-3107]: https://tegel.atlassian.net/browse/CDEP-3107?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
